feflow/protocols/protein_mutation.py

```diff
@@ -2,14 +2,51 @@
 Implementation of protein mutation protocol based on Nonequilibrium Cycling, using OpenMM as
 MD engine.
 """
+from pathlib import Path
 from typing import Dict, Any
 
 from gufe import ChemicalSystem, Context,  ProtocolUnit, AtomMapping
 
 
+def mutate_with_pdbfixer(filename: str | Path, mutation_spec: str, chain_id: str = "A", output_pdb=None):
```

Comment:
I would suggest putting this in a separate place, like a feflow.setup or feflow.utils.

Reasoning: this is something you'll do _before_ you reach the Protocol, so from a user standpoint, it's a setup tools (indeed, it might even make sense to make a separate toolling akin to kartograf or konnektor if this goes beyond just a single method!)

Reply:
Yes, 100%, this was something I added temporarily and forgot to remove. My bad!

Reply:
It shouldn't be here at all.